int16 front_center
int16 front_left
int16 front_right

int16 side_left
int16 side_right



int16 to_back
int16 to_ground




